.gitignore

```diff
@@ -18,6 +18,11 @@ distribution/openhabhome/etc/rrd4j/*.rrd
 features/*/src/main/history
 itemDump.txt
 
+#Eclipse files
```

Comment:
Please remove changes to the gitignore file. If any are (globally) necessary, they should be in a seperate PR and go into the top-level gitignore file.